packages/gatsby-graphiql-explorer/src/app/snippets.js

```diff
@@ -0,0 +1,59 @@
+const getQuery = arg => {
+  const { operationDataList } = arg
+  const { query } = operationDataList[0]
```

Comment:
I think it would be useful to do some sanity check here - if there are no queries there will "cannot read query of undefined" error happening

Reply:
I will check this again, I felt like it's handled but if not I'll put something in place

Reply:
Oh, it might be handled by code-exporter itself maybe (not sure) - let me know what happens when you have completely empty query editor ;)

Reply:
This is handled by code-exporter internally, so I'll mark this note as resolved